tidy3d/components/geometry.py

```diff
@@ -206,7 +206,7 @@ def intersections_2dbox(self, plane: Box) -> List[Shapely]:
         """
 
         # Verify 2D
-        if plane.size.count(0.0) != 1:
+        if plane.size.count(0.0) != 0:
```

Comment:
I just stumbled on this - it doesn't really make sense to me? If the goal is to verify 2D then the old version was correct. What is the issue/goal of this?

Reply:
huh that should be == 0. Lemme fix now

Reply:
The old version was giving an error when I tried to plot simulations with sim_size=(0,0,L), which had previously worked fine.